client/client-portals/components/admin/dashboard/accepted-projects/index.tsx

```diff
@@ -62,6 +64,7 @@ const AcceptedProjectDashboard = () => {
                 hasPrevPage={hasPrevPage}
                 pageNo={pageNo}
                 setPageNo={setPageNo}
+                lastPage={lastPage}
```

Comment:
Use spread operator if all the keys are same as propNames

Reply:
They are separate values. Should I create an array of all those values then use the spread operator on it?